Source/Parallelization/GuardCellManager.H

```diff
@@ -0,0 +1,42 @@
+#ifndef GUARDCELLMANAGER_H_
+#define GUARDCELLMANAGER_H_
+
+#include <AMReX_IntVect.H>
+
+class guardCellManager{
```

Comment:
Could you add a Doxygen string for this class? (Saying essentially that it holds the values of the number of guard cells required for different parts of the PIC loop)

Reply:
Sure, done.